src/main/java/uk/gov/hmcts/darts/common/service/bankholidays/Event.java

```diff
@@ -2,9 +2,13 @@
 
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+import lombok.Setter;
 
 import java.time.LocalDate;
 
+@Getter
+@Setter
 @JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
My mistake from previous ticket but as you added lombok can the fields be private?

Reply:
ACCEPT